src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -251,5 +233,76 @@ canvas {
 }
 
 .asset-error {
+    font-size: 85%;
+    font-weight: 500;
     margin: auto;
 }
+
+.badge-visited, .badge-tagged {
```

Comment:
is it a CSS thing to not just call this `.badge` and have the `tagged` and `visisted` inherited from it?

Reply:
There is already a `badge` class in bootstrap that we are leveraging and bot of these classes have the same settings.